src/core/crypto/multiKey.ts

```diff
@@ -218,7 +218,7 @@ export class MultiKey extends AccountPublicKey {
     if (index !== -1) {
       return index;
     }
-    throw new Error("Public key not found in MultiKey");
+    throw new Error(`Public key ${publicKey} not found in MultiKey ${this.publicKeys}`);
```

Comment:
Do you know what does `this.publicKeys` serialize to?
Could possibly be `[Object object]`